nodejs/README.md

```diff
@@ -1,42 +1,31 @@
-Intro
----
+# Node CSFLE Example
 
-This is a guide for ...
-https://docs.mongodb.com/ecosystem/use-cases/client-side-field-level-encryption-guide/
+## Steps
 
-Requirements
----
+1. Start a locally running `mongod` version >= 4.2 running on port 27017
+2. Install dependencies with npm
 
-Similar to CSFLE guide requirements w/additional dependencies
+   ```js
+   npm install
+   ```
 
-MongoDB Server 4.2 Enterprise
--mongocryptd
-something similar to:
-The 4.2-compatible drivers use the Enterprise-only mongocryptd process to parse the JSON schema and apply the encryption rules when reading or writing documents. Automatic encryption requires the mongocryptd process.
+3. Run the `make-data-key.js` script to make a data key. If there is an
```

Comment:
Could you add some instruction on what to do with the output of the script so the user knows what action to take before running client.js?